authority/voting/server/server.go

```diff
@@ -187,6 +189,12 @@ func (s *Server) halt() {
 func New(cfg *config.Config) (*Server, error) {
 	s := new(Server)
 	s.cfg = cfg
+	s.geo = cfg.SphinxGeometry
```

Comment:
Hmm, are there cases where we don't just want to do `s.cfg.SphinxGeometry` ? maybe `Server` doesn't need its own copy of the geometry?

Reply:
yeah we can move the geometry reference to the config if you think that is cleaner